graphql/resolvers.go

```diff
@@ -1002,7 +1002,7 @@ func (r *mutationResolver) AbortTask(ctx context.Context, taskID string) (*restM
 		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error aborting task %s: %s", taskID, err.Error()))
 	}
 	if t.Requester == evergreen.MergeTestRequester {
-		_, err = commitqueue.RemoveCommitQueueItemForVersion(t.Project, p.CommitQueue.PatchType, t.Version)
+		_, err = commitqueue.RemoveCommitQueueItemForVersion(t.Project, p.CommitQueue.PatchType, t.Version, gimlet.GetUser(ctx).DisplayName())
```

Comment:
since this is also called at line 1000, you can just set this to a variable so you don't have to get the user twice

Reply:
Good catch, thank you!